src/main/webapp/dev/js/main/studentProfile.js

```diff
@@ -84,6 +92,10 @@ $(document).ready(() => {
         finaliseEditPictureForm();
     });
 
+    $('#deletePhoto').on('click', () => {
```

Comment:
For an action as dangerous as deletion, going straight without warning is not a pleasant user experience. Should give a confirmation e.g. via modal before proceeding.